src/services/user/hooks/handle-facebook-create.js

```diff
@@ -0,0 +1,52 @@
+
+const handleFacebookCreate = () => (hook) => {
+  // Pass through this hook if there's no facebook data
+  if (!hook.data.facebookId
+    || !hook.data.facebook
+    || !hook.data.facebook.email
+    || (!hook.data.facebook.name
+      && !hook.data.facebook.first_name
+      && !hook.data.facebook.last_name)
+    ) {
+    return hook;
+  }
+
+  const userService = hook.app.service('/users');
+
+  // Check whether a user with same facebook email exists
+  return userService.find({ query: { email: hook.data.facebook.email } })
+    .then(results => {
+      // Handle answer in both array and object forms
+      const existingUser = results[0] || (results.data && results.data[0]);
+
+      // User properties to be created or updated
+      const data = {
+        facebookId: hook.data.facebookId,
+        email: hook.data.facebook.email,
+        name: hook.data.facebook.name ||
+          `${hook.data.facebook.first_name} ${hook.data.facebook.last_name}`,
+      };
+
+      // Patch existing user's properties
+      if (existingUser) {
+        return userService.patch(existingUser._id, data) // eslint-disable-line no-underscore-dangle
+          .then(updatedUser => {
+            // Set `hook.result` to skip the actual `create` since we updated it already
+            hook.result = updatedUser; // eslint-disable-line no-param-reassign
```

Comment:
So, from that code, it will try create a user after you create a facebook user...

Then, it should return error or sth. or have a new user beside what you have created...
Have you seen any kind of these behaviors?